sources/packages/api/src/utilities/date-utils.ts

```diff
@@ -15,6 +15,8 @@ dayjs.extend(dayOfYear);
 export const DATE_ONLY_ISO_FORMAT = "YYYY-MM-DD";
 export const DATE_ONLY_FORMAT = "YYYY MMM DD";
 export const DATE_TIME_FORMAT = "YYYY-MM-DD HH:mm:ss";
+export const TIMESTAMP_CONTINUOUS_FORMAT = "YYYYMMDD-HHmmss";
```

Comment:
IMO when we put year/month/day altogether it is tricky to read, can we just add a simple separator like `YYYY-MM-DD-HH_mm_ss`?